app/src/main/java/org/p2p/wallet/alarmlogger/model/AlarmSendErrorConverter.kt

```diff
@@ -235,36 +168,4 @@ class AlarmSendErrorConverter(
             payload = gson.toJson(request)
         )
     }
-
-    fun toWeb3ErrorRequest(web3Error: String): AlarmErrorsRequest {
-        // no user public key here, because we don't have it at this step
-        val request = AlarmErrorsWeb3Request(
-            web3Error = web3Error
-        )
-
-        return AlarmErrorsRequest(
-            logsTitle = "Web3 Registration Android Alarm",
-            payload = gson.toJson(request)
-        )
-    }
-}
-
-private fun Throwable.getSimulationError(): String = when (this) {
-    is SimulationException -> "Simulation error: ${getDirectMessage().orEmpty()}"
-    else -> emptyString()
-}
-
-private fun Throwable.getFeeRelayerError(): String = when (this) {
-    is FeeRelayerException -> "FeeRelayer error: ${getDirectMessage().orEmpty()}"
-    else -> emptyString()
-}
-
-private fun Throwable.getBlockchainError(): String = when (this) {
```

Comment:
this formatting logic is extracted to a separate throwable formatter special for such cases